chitterApi.js

```diff
@@ -0,0 +1,48 @@
+class ChitterApi {
+
+  loadPeeps(callback) {
+    const callbackFn = callback || (() => { })
```

Comment:
You reference this a lot - not sure why you have it but you could make it a line in the constructor eg. ```this.callbackFn()``` instead of writing it every time